{"uri":"file:///home/tarcisio/dev/phpfin/vendor/symfony/config/Definition/Exception/Exception.php","root":{"kind":0,"name":"","children":[{"kind":512,"name":"Symfony\\Component\\Config\\Definition\\Exception","location":[11,0,11,56]},{"kind":1,"name":"Symfony\\Component\\Config\\Definition\\Exception\\Exception","location":[18,0,20,1],"associated":[{"kind":1,"name":"RuntimeException"}],"description":"Base exception for all configuration exceptions."}]}}